src/plugins/intel_gna/src/gna_plugin_config.cpp

```diff
@@ -46,6 +46,39 @@ static const caseless_unordered_map<std::string, std::pair<Gna2AccelerationMode,
 };
 OPENVINO_SUPPRESS_DEPRECATED_END
 
+/**
+ * @brief Parse string into value without throwing
+ * @param value A parsed value
+ * @param def_value A value to return in case of exception
+ * @return parsed value, default value otherwise
+ * @ingroup ov_runtime_gna_prop_cpp_api
+ */
+template <class T>
+static T FromString(const std::string& value, T def_value) {
+    T result = def_value;
+    try {
+        result = ov::util::from_string<T>(value);
+    } catch (std::exception&) {
+    }
```

Comment:
do we want to log error?

Reply:
The purpose of this method is to catch the eventual exception from ov::util::from_string without raising any errors - this is why the def_value is provided as a parameter. 
log::warning() could be useful here, however.